demos/python_demos/object_detection_demo/object_detection_demo.py

```diff
@@ -171,10 +171,11 @@ def get_plugin_configs(device, num_streams, num_threads):
     return config_user_specified
 
 
-def draw_detections(frame, detections, palette, labels, threshold, draw_landmarks=False):
+def draw_detections(frame, detections, palette, labels, threshold):
     size = frame.shape[:2]
     for detection in detections:
         if detection.score > threshold:
+            draw_landmarks = isinstance(detection, DetectionWithLandmarks)
```

Comment:
Seems like an unnecessary variable, IMO. You could just inline it in the `if` where it's used.

Reply:
Removed